include/internal/kmgmt_util.h

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2020 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#include "internal/param_build.h"
+#include <openssl/safestack.h>
+
+int ossl_kmgmt_params_set_int(OSSL_PARAM_BLD *bld, OSSL_PARAM *p,
+                              const char *key, int num);
```

Comment:
Re the names, there's nothing particularly keymgmt about these functions...

Reply:
Nup but their use is pretty specifically related to keymanagement..
Where do you think they belong? Or what namespace should it use?

Reply:
They look like fairly general ossl_param routines to me

Reply:
so we already have OSSL_PARAM_set_int(), and I didnt want to get these confused..
I put them in keymgmt because I thought they were probably only going to be used by that subsystem..

Reply:
It's assumptions like that that have us end up with a generic URL parser as an OCSP function....
Since this is centered around OSSL_PARAM_BLD, that may be a more usable prefix?

Reply:
moved